WalletWasabi.Tests/UnitTests/IoTests.cs

```diff
@@ -147,16 +127,19 @@ static bool IsStringArraysEqual(string[] lines1, string[] lines2)
 			var fileCount = Directory.EnumerateFiles(Path.GetDirectoryName(ioman1.FilePath)).Count();
 			Assert.Equal(0, fileCount);
 
-			// Check Mutex usage on simultaneous file writes.
+			// Check AsyncLock usage on simultaneous file writes.
 
-			DigestableSafeMutexIoManager ioman2 = new DigestableSafeMutexIoManager(file2);
+			DigestableIoManager ioman2 = new DigestableIoManager(file2);
+
+			AsyncLock asyncLock1 = new AsyncLock();
+			AsyncLock asyncLock2 = new AsyncLock();
 
 			await Task.Run(async () =>
 			{
-				using (await ioman1.Mutex.LockAsync())
+				using (await asyncLock1.LockAsync())
 				{
-					// Should not be a problem because they use different Mutexes.
-					using (await ioman2.Mutex.LockAsync())
+					// Should not be a problem because they use different AsyncLock.
+					using (await asyncLock2.LockAsync())
```

Comment:
Why do we need these two locks?

Reply:
I was mimicking the original pattern.

Reply:
The test makes no sense now, I removed it.